test/functional/feature_config_args.py

```diff
@@ -93,8 +93,8 @@ def test_args_log(self):
                     'Command-line arg: rpcpassword=****',
                     'Command-line arg: rpcuser=****',
                     'Command-line arg: torpassword=****',
-                    'Config file arg: regtest="1"',
-                    'Config file arg: [regtest] server="1"',
+                    'Config file arg: %s="1"' % self.chain,
```

Comment:
this could have also been 'Config file arg: chain=%s=' % self.chain, but this works too.

Reply:
This wouldn't work, as the fundamental content of the config file is not written by this test itself right now at this stage, but by the function `util.initialize_datadir()` in the course of setting up the `BitcoinTestFramework`, which uses the `chainname=1` format:
https://github.com/bitcoin/bitcoin/blob/23fab1a3dfe6f308880197dbe2bd7ce70866f13c/test/functional/test_framework/util.py#L311-L313